Source/player.cpp

```diff
@@ -547,23 +547,20 @@ void RespawnDeadItem(Item *itm, Point target)
 
 	dItem[target.x][target.y] = ii + 1;
 
-	Items[ii] = *itm;
+	Items[ii] = itm;
 	Items[ii].position = target;
-	RespawnItem(&Items[ii], true);
-
-	itm->_itype = ItemType::None;

```

Comment:
It feels like luck that the original code actually worked :D

Could you move `NetSendCmdPItem` in here, I think that would make the code a lot more sensible and less fragile.